test/models/clinic_test.rb

```diff
@@ -75,11 +75,10 @@ class ClinicTest < ActiveSupport::TestCase
 
   describe 'callbacks' do
     describe 'updating coordinates when address changes' do
-      before { @cached_api_key = ENV['GOOGLE_GEO_API_KEY'] }
-      after { ENV['GOOGLE_GEO_API_KEY'] = @cached_api_key }
+      before { Geokit::Geocoders::GoogleGeocoder.api_key = '123' }
+      after { Geokit::Geocoders::GoogleGeocoder.api_key = nil }
```

Comment:
By my read, at a high level this test has changed rather than just updated - is that the intent? 
Before we stashed and restored the value - now we set it and then unset it.

Reply:
this is a great flag! Yeah, this is an intentional change -- we handle api key setting better in the initializer now, and so we don't have to do a dance with env vars, but because this is a class attribute we want to reset it to the assumed state it was in. It's essentially the same dance of 'reset it to the state it was in' but we're not reading straight from the env var in the locator code anymore.